### PR TITLE
Window popup

### PR DESCRIPTION
**Pull request by Tracy-miranja**
I implemented the following interactions:
When the user clicks the button to check project details, the popup with details about the project appears.
When the user clicks the close (X) button, the popup disappears.
I use a JavaScript array to store all of the information for all projects.
I stored the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, and link to the source.
I updated the main page so that the projects section is created dynamically

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const modalData = [
...
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
```
Comment (iambenkis) [inline popup.js]: - [ ] Good job adding the pop-up, in your web page :clap: , However I noticed that as the requirements said that you should stick as much as possible to the Figma design , the pop-up should be above the main section :+1: , I noticed for you that the pop-up is under the navbar, Kindly fix in your desktop version :+1:  

| Figma design | Your design |
| :---: | :---: |
| Screenshot from 2023-02-04 02-02-24 | Screenshot from 2023-02-04 02-11-47 |

File: popup.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ const modalData = [
...
     tech: ['HTML', 'CSS', 'Javascript', 'github', 'ruby', 'Bootstraps'],
     button: {
       link: 'https://example.com/',
```
Comment (iambenkis) [inline popup.js]: - [ ] Good job adding the pop-up, in your web page :clap: , However I noticed that as the requirements said that you should stick as much as possible to the Figma design , the pop-up should be above the main section :+1: , I noticed for you that the pop-up is under the navbar, Kindly fix in your mobile version :+1:  

| Figma design | Your design |
| :---: | :---: |
| Screenshot from 2023-02-04 02-03-16 | Screenshot from 2023-02-04 02-20-14 |

Comment (iambenkis) [inline popup.js]: - [ ] Kindly make sure that this icon in the button `see project` does not provide a broken link :+1: 
Screenshot from 2023-02-04 02-03-57

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Tracy-miranja 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: popup.js (hunk 2 above)
Comment (Shedrack-Sunday) [inline popup.js]: - [ ] Just as mentioned by the previous reviewer, the source link has a broken image tag here. Your whole code was commented, so I suggested you trace the icon or image link to get that fixed 👍 

# View:
image

# Figma view:
image

File: popup.js (hunk 1 above)
Comment (Shedrack-Sunday) [inline popup.js]: - [ ] Also as suggested by the previous reviewer and rightly so, the buttons for your live and source links should display without an overflow to scroll down. This is in line with best practices for web accessibility and gives the best user experience 👏 
- Seeing that your design is great, I suggest you reduce the content of the `loresum` wording and possible overflow, so the buttons show on the first display. good job so far 👍 

# View now:
image

# Figma reommnndation :
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Tracy-miranja      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯 



Highlights
✅All linters are checked
✅ Descriptive commit messages 
  
Required Changes ♻️

- [x] You did make efforts in the  Project and you really need a great job. However, there seem to be a few adjustments needed on the mobile view pop-up to make your whole code stand out and look good. 👍 

- [x] the Reason for taking web accessibility seriously onscreen sizes is because it ensures that users with varying devices, including smaller screens such as smartphones, are able to access and use a website effectively. This includes individuals with disabilities who may use assistive technologies such as screen readers or magnifiers to access the web. By ensuring that websites are accessible on smaller viewports, developers can help make the web more inclusive for all users.  Kindly fix the margin on the right of the screen view. good job so far 👏 


_check the comment section for a detailed review_

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by frank1738 — COMMENTED** [review]
Hi @Tracy-miranja , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with failing linter checks. Kindly fix that and resubmit your oproject

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by binhussen — APPROVED** [review]
Hi @Tracy-miranja , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

- [Optional] Kindly you added unnecessary code in your readme file below the license section 👌👌Please Remove it to have clean and professional readme 👍👌

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**